### ISSUE TITLE
Installation issues (RBAC?)

### ISSUE THREAD

**Issue by cv** [sorenmat/k8s-rds#22]
Hi!

I managed to work around https://github.com/sorenmat/k8s-rds/issues/21 by running `go build` and creating a docker image locally and pushing it to ECR so that Kubernetes could see it.

I have RBAC enabled in my cluster, and it looks like the permissions system isn't letting the `k8s-rds` binary do its thing upon startup:

```
$ kubectl logs -lname=k8s-rds
2018/03/29 23:43:45 Starting k8s-rds
panic: customresourcedefinitions.apiextensions.k8s.io is forbidden: User "system:serviceaccount:default:default" cannot create customresourcedefinitions.apiextensions.k8s.io at the cluster scope

goroutine 1 [running]:
main.main()
	/Users/carlosvillela/go/src/github.com/sorenmat/k8s-rds/main.go:160 +0x651
```

What's the simplest way to solve this?

(I volunteer to document it accordingly in a follow-up PR once we get it working, by the way)

**Comment by cv** [comment]
This is by no means acceptable from a security standpoint, but did get me past that error:

```bash
kubectl create clusterrolebinding \
  rds-admin-binding \
  --clusterrole=cluster-admin\
  --user=system:serviceaccount:default:defaultusterrolebinding \
  --clusterrole=cluster-admin \
  --user=system:serviceaccount \
  rds-admin-binding
```

It creates the following `ClusterRoleBinding`:

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: rds-admin-binding
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: cluster-admin
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: User
  name: system:serviceaccount:default:default
```

**Comment by sorenmat** [comment]
Right now the pod does need quite a bit of access..
I assume something like this needs to be configure https://github.com/coreos/etcd-operator/blob/master/doc/user/rbac.md

It basically need access to create the CustomResourceDefinitions to extend the cluster.
PR are more then welcome :)

**Comment by HerrmannHinz** [comment]
@cv i can confirm that the RBAC hack got it working indeed. seems like u have some typos in your yaml stanza, following worked for me: 

`kubectl create clusterrolebinding \
  --clusterrole=cluster-admin\
  --user=system:serviceaccount:default:default \
  --clusterrole=cluster-admin \
  --user=system:serviceaccount \
  rds-admin-binding`

@sorenmat good work man! nice project. i'll try to help a little bit :)

**Comment by clook** [comment]
Hi here, thanks @sorenmat for the work!
Rather than using the `cluster-admin` ClusterRole, I created a dedicated ClusterRole that is powerful enough to get the thing working (may still require some tunings):
```

**Comment by sorenmat** [comment]
closed by PR #45

**Comment by matthiasak** [comment]
@cv good stuff, your comments above helped!
